lib/syskit/roby_app/remote_processes/process.rb

```diff
@@ -42,12 +42,6 @@ def dead!
                     @alive = false
                 end
 
-                # Returns the task context object for the process' task that has this
-                # name
-                def task(task_name)
-                    process_client.name_service.get(task_name, process: self)
-                end
-
                 # Cleanly stop the process
```

Comment:
We cant be sure if the IOR is known at this time, I removed this method because the one implemented at the base class is better suited to treat any errors related to not knowing the IOR.